content/using/install.md

```diff
@@ -77,11 +77,17 @@ http: live (insecure, loopback) on 12321
 ~sampel_marzod:dojo>
 ```
 
-When your planet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).
+When your comet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).
```

Comment:
Will you ever see the `chat-cli` prompt on boot? In the interest of simplicity we should probably remove the `ctrl-x` stuff from the initial installation

Reply:
I agree if true.

Reply:
I checked, and booting to chat cli is a bug that has been fixed.  I have simplified the language here as a result.